### DISCUSSION TITLE
Is there a workaround for this?

### DISCUSSION THREAD

**Discussion by AstroBear64** [valinet/ExplorerPatcher#2242] [Q&A]
![Screenshot 2023-10-04 124658](https://github.com/valinet/ExplorerPatcher/assets/146986881/a7e2ce1e-e999-43bc-b6c8-5f7608ccc7a0)
Seems like a similar problem to the 'pin to start' issue. Is there and way to change this menu? The 'personalize this list' button directs to Windows 11 settings and the changes are not applied there.

**Comment by vippium** [comment]
It is fixed in latest pre release version
